resources/fileTemplates/internal/Magento Module Controller Backend Class.php.ft

```diff
@@ -0,0 +1,31 @@
+<?php
+#parse("PHP File Header.php")
+#if (${NAMESPACE})
+
+namespace ${NAMESPACE};
+#end
+
+#set ($uses = ${USES})
+#foreach ($use in $uses.split(","))
+use $use;
+#end
+
+class ${NAME}#if (${EXTENDS}) extends ${EXTENDS}#end#if (${IMPLEMENTS}) implements ${IMPLEMENTS}#end {
+#if (${ACL})
+    /**
+     * Authorization level of a basic admin session
+     */
+    const ADMIN_RESOURCE = '${ACL}';
+
+#end
+    /**
+     * Execute action based on request and return result
+     *
+     * @return ResultInterface|ResponseInterface
+     * @throws NotFoundException
```

Comment:
Is there any reason of throwing `NotFoundException ` at the moment? 🤔

Reply:
Yes, all controllers implement [ActionInterface](https://github.com/magento/magento2/blob/2.3/lib/internal/Magento/Framework/App/ActionInterface.php) and the execute method of action interface throws `NotFoundException`.
Code reference: https://github.com/magento/magento2/blob/2.3/lib/internal/Magento/Framework/App/ActionInterface.php#L32